misc/sample_application/rtvm/Makefile

```diff
@@ -6,7 +6,7 @@ CFLAGS = -Wall -Wextra -Wabi=11 -pedantic
 LDLIBS = -lrt -pthread
 
 all: rtApp.c
-	$(CC) $(CFLAGS) -o rtapp rtApp.c ivshmemlib.c
+	$(CC) $(CFLAGS) -o ../build/rtapp rtApp.c ivshmemlib.c
```

Comment:
Should be using `$(OUT_DIR)/rtapp` here instead I guess?

Reply:
1 . Modify the Makefile in the parent directory pass the OUT_DIR to the sub-makefies.
2. Modify the Makefile in the sub-directory in case the user just run the make command in the sub-directory.
Fore update the commit in this PR